docs/rules/README.md

```diff
@@ -52,6 +53,7 @@ These rules warn of possible errors that can occur at `terraform apply`. Rules m
 These rules suggest to better ways.
 
 - [aws_instance_previous_type](aws_instance_previous_type.md)
+- [aws_db_instance_invalid_engine](aws_db_instance_invalid_engine.md)
```

Comment:
It's listed above as possible errors, so we don't need to list it here.

Reply:
Addressed. https://github.com/terraform-linters/tflint-ruleset-aws/pull/61/commits/1a8a65629aaec98305a8f14438051660ff0d1f55